tests/performance_tests/log_reader.py

```diff
@@ -412,13 +470,19 @@ def calcAndReport(data: chainData, tpsTestConfig: TpsTestConfig, artifacts: Arti
     notFound = []
     populateTrxSentTimestamp(trxSent, trxDict, notFound)
 
+    prodDict = {}
+    getProductionWindows(prodDict, blockDict, data)
+
     if len(notFound) > 0:
         print(f"Transactions logged as sent but NOT FOUND in block!! lost {len(notFound)} out of {len(trxSent)}")
+        if argsDict.get("printMissingTransactions"):
+            print(notFound)
```

Comment:
It would be far too verbose. In a proper testing run resulting in failure this would be thousands or tends of thousands of ids added to the report. It's the reason we only printed the length of the failures in the past. It used to spam the CI with all those missing transactions iirc.